frontend/public/locales/zh/build.json

```diff
@@ -0,0 +1,27 @@
+{
+  "Rebuild": "重建",
+  "Cancel build": "取消构建",
+  "Are you sure you want to cancel this build?": "您确定要取消此构建吗？",
+  "Yes, cancel": "是，取消",
+  "No, don't cancel": "否，不要取消",
+  "View logs": "查看日志",
+  "Memory usage": "内存用量",
+  "CPU usage": "CPU 用量",
+  "Filesystem": "Filesystem",
+  "Pipeline build strategy deprecation": "管道构建策略过时",
+  "pipelineBuildStrategyAlert": "pipelineBuildStrategyAlert",
+  "OpenShift Pipelines based on Tekton": "基于 Tekton 的 OpenShift Pipelines",
+  "Try the OpenShift Pipelines tutorial": "试用 OpenShift Pipelines 指南",
+  "Build details": "构建详情",
+  "Triggered by": "触发者",
+  "Started": "Started",
```

Comment:
Same as other "Started"

Reply:
This has not been updated.

Reply:
I'm going to manually drop in the translation in the other file here so we don't have to send this back.